.rubocop.yml

```diff
@@ -1,6 +1,8 @@
 # Rubocop is wrong about these styles.
 Style/EmptyMethod:
   Enabled: False
+Style/RaiseArgs:
+  EnforcedStyle: compact
```

Comment:
There is a deeper reason for the rubocop default:
If you need to pass a backtrace for some reason, `raise Klass.new('message')` will need to become `raise Klass.new('message'), 'message', backtrace`. Thus, it has become convention to do:
`raise Klass, 'message'` which extends to the nicer looking `raise Klass, 'message', backtrace`

Once again, it might be a good idea to adhere to rubcop standards first. Why? Consider [Shuhari](https://en.wikipedia.org/wiki/Shuhari) from martial arts:
    shu (守) "protect", "obey"—traditional wisdom—learning fundamentals, techniques, heuristics, proverbs
    ha (破) "detach", "digress"—breaking with tradition—detachment from the illusions of self
    ri (離) "leave", "separate"—transcendence—there are no techniques or proverbs, all moves are natural, becoming one with spirit alone without clinging to forms; transcending the physical

I'd strongly suggest to obey and learn "traditional wisdom" before breaking with tradition.

Reply:
The reason that I decided to enforce the `compact` style is that it reminds me that this is an **instance** of the `ActiveModel::Errors`. I find the alternative syntactic style _magical_ - in a negative sense. It sort of looks like a message is being passed to a Class - beziehungsweise to a Class method, no?

There are many beautiful elements to the Eastern traditions. Yet it was the West whose daring, experimentation, and _rule-**breaking**_ catapulted the _invention_ of airplanes, pianos, computers and the _discovery_ of spacetime. Tradition _can_ have wisdom. But we must actively probe, test, attack it to find if it is true - still true, was always true.

> Verloren sei uns der Tag, wo nicht ein Mal getanzt wurde! Und falsch heiße uns jede Wahrheit, bei der es nicht ein Gelächter gab!
--- Friedrich Nietzsche, _Also Sprach Zarathustra_, III Buch, "Von alten und neuen Tafeln"

Reply:
Be that as it may. I'd strongly suggest to obey and learn and actively use most of "traditional wisdom" _before_ breaking with tradition.   

It is not about this individual rule, but more about the pattern to deviate from the current perception of idiomatic code before learning current idiomatic code.

Whatever you currently perceive as "wrong" and decide to do differently might make it harder for others to read and understand your code – I really don't think you have a deep enough understanding yet to make those calls.

Writing code so that it works (for you) is the easy bit. Writing it so reading and understanding is easy for others is the hard bit.

<hr/>

On a meta note, this is the last time I will engage in a discussion like this. I have stated my opinion on these matters, and I am getting annoyed as you can probably tell.

I am trying to motivate why certain idioms are "standard", but I do not want to engage in a time-consuming personal flavour discourse about them.

<hr/>

Back to topic (for the last time):
- `raise` and `rescue` are like opposite twins: you `raise StandardError` and you `rescue StandardError`
- `ActiveModel::Errors` is a part of the Rails framework, while`raise Something` is a plain Ruby construct, not sure I get the point here.
- Calling `Error.new('message')` is quite literally passing `'message'` to the class method `.new` on `Error`. Not sure what point you're trying to make here either.

I do understand the sentiment that it secretly constructs an instance, but I don't think it's too hard to get what's going on here.